ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -38,6 +38,8 @@ import qualified Cardano.Ledger.Shelley.Constraints as SL
 import           Ouroboros.Consensus.Shelley.Protocol.Crypto (StandardCrypto)
 import qualified Shelley.Spec.Ledger.API as SL
 import qualified Shelley.Spec.Ledger.BaseTypes as SL
+import qualified Shelley.Spec.Ledger.Tx as SL
+import qualified Cardano.Ledger.Core as Core
```

Comment:
`Cardano.Ledger.Core` seems to already be in scope with qualified name `LC`.

Reply:
Yeah, nice catch. I've removed `Tx`